local/gugcat/classes/form/editgradeform.php

```diff
@@ -0,0 +1,70 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * A moodleform allowing the editing of the grade options for an individual grade item
+ *
+ * @package    local_gugcat
+ * @copyright  2020
+ * @author     Accenture
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+require_once("$CFG->libdir/formslib.php");
+require_once($CFG->dirroot . '/local/gugcat/locallib.php');
+class editgradeform extends moodleform {
```

Comment:
As much as possible, please reuse the addgradeform.php,, I suggest rename it to addeditgradeform to make it generic for both adding and updating grades.